force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +90,17 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
-
+       List<String> words = sentence.split(' ');
         // Create a set to store the unique words
-
+        Set<String> wordsOfSentence = new Set<String>();
         // Add each word to the set
-
+        for(String word :words){
+            wordsOfSentence.add(word);
+        }
         // Join the unique words back into a sentence
-
+        String result = String.join(new List<String>(wordsOfSentence), ' ');
         // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+        return result; // Replace null with the variable you used to store the result
```

Comment:
There is no need to iterate over a list. You can just add it directly to the Set. 

```
  public static String removeDuplicateWords(String sentence) {
        Set<String> uniqueWords = new Set<String>(sentence.split(' '));
        return String.join(uniqueWords, ' ');
    }
    ```